packages/shared/components/drawerContent/AccountCreation.svelte

```diff
@@ -0,0 +1,110 @@
+<script lang="typescript">
+    import { Button, ColorPicker, Input, Spinner, Text } from 'shared/components'
+    import { getTrimmedLength } from 'shared/lib/helpers'
+    import { localize } from '@core/i18n'
+    import { showAppNotification } from 'shared/lib/notifications'
+    import { closePopup, popupState } from 'shared/lib/popup'
+    import { AccountColors, MAX_ACCOUNT_NAME_LENGTH, setSelectedAccount, wallet } from 'shared/lib/wallet'
+
+    export let error = ''
+    export let onCreate = (..._: any[]): void => {}
+    export let onCancel = (..._: any[]): void => {}
```

Comment:
why not just `(): void => {}` to define the type? It's looks ugly and confusing imo, what is the difference?

Reply:
Because that means you create a default implementation for it that does nothing. If  you need arguments, you should to type it as follows: 

`onCancel: (..._: any[]) => void)`

The difference is that if you omit the `..._: any[]`, but you would pass a function with arguments as a prop to the component, the TS type system would complain, since it expected to receive a function without arguments.

It would be better to define + type the arguments that `onCancel` can receive, but this might be difficult if the component is used at multiple places, with different arguments.